api.raml

```diff
@@ -0,0 +1,107 @@
+#%RAML 1.0
+title: ID Broker API
+protocols: [ HTTPS ]
```

Comment:
May need to say http and https so in test mode it can just listen on http, especially since the container only listens on http/80 and https will be handled by the load balancer.